modules/storages/config/locales/en.yml

```
@@ -39,6 +39,8 @@ en:
       setting_up_storages: "For setting up file storages, please visit"
       setting_up_storages_non_admin: "Administrators can set up file storages in Administration / File Storages."
       all_available_storages_already_added: "All available storages are already added to the project."
+      oauth_client_id: "OAuth Client ID from attached storage (Nextcloud) that allows OpenProject to connect to the storage. This should be a lengthy string of garbled characters. Please see the storage manufacturers instructions on how to setup an OAuth application."
```

The translation string here is Nextcloud specific but the field itself is not necessarily Nextcloud specific.
```suggestion
      oauth_client_id: "OAuth Client ID for the attached storage, i.e. Nextcloud, that allows to identify OpenProject when connecting to the storage. This should be a lengthy string of garbled characters. For obtaining the OAuth Client ID, please see the storage manufacturers instructions on how to setup an OAuth application."
```